src/components/QuickGrid/QuickGrid.Props.ts

```diff
@@ -6,23 +6,31 @@ export interface IQuickGridProps {
     groupBy: Array<string>;
     gridClassName?: string;
     headerClassName?: string;
-    sortColumn?: string;
-    sortDirection?: 'ASC' | 'DESC';
     rowHeight: number | ((info: { index: number }) => number); // Number or a function that returns the height of a row given its index
     headerHeight: number;
     overscanRowCount?: number;
-    highlightHoverRow?: boolean;
     onSelectedRowChanged?: (selectedRowIndex: number) => void;
     onRowDoubleClicked?: (row: any) => void;
+    displayGroupContainer?: boolean;
+
+    sortColumn?: string;
+    sortDirection?: 'ASC' | 'DESC';
+    onGroupByChanged?: (groupBy: Array<string>) => void;
+
+    groupBySortColumn?: string;
```

Comment:
The grid now supports multiple levels of grouping, you can sort by one group and column